Command/HealthCheckCommand.php

```diff
@@ -36,13 +43,34 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $checkName = $input->getArgument('checkName');
         $runner = $this->getContainer()->get('liip_monitor.runner');
-        $runner->addReporter(new ConsoleReporter($output));
+
+        if ($input->getOption('nagios')) {
+            $runner->addReporter(new RawConsoleReporter($output));
```

Comment:
Raw console reporter just outputs like console reporter but without the console coloring.